lib/src/argument_list_visitor.dart

```diff
@@ -504,7 +504,9 @@ class ArgumentSublist {
     // TODO(rnystrom): Should we step into parenthesized expressions?
 
     if (expression is ListLiteral) return expression.leftBracket;
-    if (expression is MapLiteral) return expression.leftBracket;
+    // TODO(rnystrom): should we return expression.leftBracket for sets as well?
+    if (expression is SetOrMapLiteral && !expression.isSet)
```

Comment:
Yes, just remove the `&& !expression.isSet` part and allow this to match sets too. (The fact that it doesn't was an oversight.)